client/src/cordova/apple/OutlineAppleLib/Package.swift

```diff
@@ -24,7 +24,7 @@ let package = Package(
         // CocoaLumberjack 3.8.0 dropped support for iOS < 11 and macOS < 10.13.
         // See https://github.com/CocoaLumberjack/CocoaLumberjack/releases/tag/3.8.0.
         // These cannot be upgraded without also upgrading the entire project.
-       .iOS(.v13),
+       .iOS("15.5"),
```

Comment:
I think you can also use `.15_5`, but I guess it doesn't matter.

Reply:
That's what I thought but you can't. I had it in a previous commit, but it failed to build:

```
  /Users/runner/work/outline-apps/outline-apps/client/src/cordova/apple/OutlineAppleLib/Package.swift:28:14: error: reference to member 'v15_5' cannot be resolved without a contextual type
         .iOS(.v15_5),
```

Turns out only 15 exists: http://go/appledoc/packagedescription/supportedplatform/iosversion/v15. I guess only the 10_X versions specified more granular version types.